Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -174,12 +158,18 @@ private void HeaderClicked(ObjectsTabHeader.Header header)
         RefreshObjectList();
     }
 
+    private void RefreshButtonPressed(BaseButton.ButtonEventArgs args)
+    {
+        RefreshObjectList();
+    }
+
     private enum ObjectsTabSelection
     {
         Grids,
         Maps,
-        Stations,
+        Stations
```

Comment:
```suggestion
        Stations,
```
Leave the trailing comma. It makes makes future diffs cleaner when someone adds a value to the enum.

Reply:
Should I turn these on for the project style settings then?
![image](https://github.com/space-wizards/space-station-14/assets/47093363/a9430d59-7441-4db0-bf5a-1fea6128f4c6)

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/47093363/84509e64-3b52-43aa-9599-e43c2b611346)
Will wait to push this one.

Reply:
Put it in another PR. For multiline lists yes. For singleline doesn't matter as much.